independent-projects/tools/message-writer/src/main/java/io/quarkus/devtools/messagewriter/MessageIcons.java

```diff
@@ -1,14 +1,23 @@
 package io.quarkus.devtools.messagewriter;
 
-public final class MessageIcons {
+import io.smallrye.common.os.OS;
 
-    public static final String OK_ICON = "\u2705";
-    public static final String NOK_ICON = "\u274c";
-    public static final String NOOP_ICON = "\uD83D\uDC4D";
-    public static final String WARN_ICON = "\uD83D\uDD25";
-    public static final String ERROR_ICON = "\u2757";
+public enum MessageIcons {
 
-    private MessageIcons() {
+    OK_ICON("\u2705"),
+    NOK_ICON("\u274c"),
+    NOOP_ICON("\uD83D\uDC4D"),
+    WARN_ICON("\uD83D\uDD25"),
+    ERROR_ICON("\u2757");
+
+    String icon;
+
+    MessageIcons(String icon) {
+        this.icon = icon;
+    }
+
+    public String getIcon() {
```

Comment:
Also make sure to override `toString()`, for cases where the constants here are concatenated into Strings

Reply:
Done